code/modules/power/supermatter/supermatter.dm

```diff
@@ -709,7 +709,20 @@
 		return
 	if(moveable && default_unfasten_wrench(user, I, time = 20))
 		return
-	if(user.drop_item())
+	if(istype(I, /obj/item/scalpel/supermatter))
+		var/obj/item/scalpel/supermatter/scalpel = I
+		to_chat(user, "<span class='notice'>You carefully begin to scrape \the [src] with \the [I]...</span>")
+		if(I.use_tool(src, user, 100, volume=100))
+			if (scalpel.usesLeft)
+				to_chat(user, "<span class='danger'>You extract a sliver from \the [src]. \The [src] begins to react violently!</span>")
+				new /obj/item/nuke_core/supermatter_sliver(drop_location())
```

Comment:
Is there a chance the sliver can be pulled into the SM like other items are?

Reply:
Supermatter shard spawns on top of the supermatter, so it should not be pulled in.